safe_transaction_service/history/models.py

```diff
@@ -509,6 +521,16 @@ def from_decoded_event(cls, event_data: EventData) -> Union["ERC721Transfer"]:
     def value(self) -> Decimal:
         return self.token_id
 
+    def to_erc20_transfer(self):
+        return ERC20Transfer(
+            ethereum_tx=self.ethereum_tx,
+            address=self.address,
+            _from=self._from,
+            to=self.to,
+            log_index=self.log_index,
+            value=self.token_id,
```

Comment:
should you be assiging `self.value` instead of `self.token_id`?

Reply:
No, as I'm converting a `ERC721` to an `ERC20` 😉